README.md

```diff
@@ -1,131 +1,166 @@
-# Aave Protocol js
+# AAVE-JS
+
+AAVE is a decentralized non-custodial liquidity market protocol where users can participate as depositors or borrowers. The AAVE Protocol is a set of open source smart contracts which facilitate the lending and borrowing of user funds. These contracts, and all user transactions/balances are stored on a public ledger called a blockchain, making them accessible to anyone.
+
+The aave-js package gives developers access to methods for formatting data and executing transactions on the AAVE protocol.
+
+1. [Quick Start](#quick-start)
+2. [Data Formatting Methods](#data-formatting-methods)
+   - a. [User Data](#user-data)
+      - [formatUserSummaryData](#formatUserSummaryData) 
+   - b. [Reserve Data](#reserve-data)
+      - [formatReserves](#formatReserves) 
+3. [Transaction Methods](#transaction-methods)
+   - a. [Lending Pool V2](#lending-pool-v2)
+      - [deposit](#deposit)
+      - [borrow](#borrow)
+      - [repay](#repay)
+      - [withdraw](#withdraw)
+      - [swapBorrowRateMode](#swapBorrowRateMode)
+      - [setUsageAsCollateral](#setUsageAsCollateral)
+      - [liquidationCall](#liquidationCall)
+      - [swapCollateral](#swapCollateral)
+      - [repayWithCollateral](#repayWithCollateral)
+   - b. [Staking](#staking)
+      - [stake](#stake)
+      - [redeem](#redeem)
+      - [cooldown](#cooldown)
+      - [claimRewards](#claimRewards)
+   - c. [Governance V2](#governancev2)
+      - [Governance](#governance)
+      - [create](#create)
+      - [cancel](#cancel)
+      - [queue](#queue)
+      - [execute](#execute)
+      - [submitVote](#submitVote)
+      - [GovernanceDelegation](#governanceDelegation)
+      - [delegate](#delegate)
+      - [delegateByType](#delegateByType)
+   - d. [Faucets](#faucets)
+      - [mint](#mint)
+4. [Lint](#lint)
+5. [Build](#build)
 
-Aave protocol logic package, supports main and uniswap markets
 
-The simplest way to get initial protocol data - use our subgraphs on TheGraph protocol:
-https://thegraph.com/explorer/subgraph/aave/protocol-multy-ropsten-raw - ropsten
-https://thegraph.com/explorer/subgraph/aave/protocol-multy-kovan-raw - kovan
-https://thegraph.com/explorer/subgraph/aave/protocol-multy-raw - mainnet
+# Quick Start
+
+This package uses [ethers v5](https://github.com/ethers-io/ethers.js#readme) as peer dependency, so make sure you have installed it in your project.
 
-_/graphql/_ folder contains graphql documents with the structures needed for aggregation methods.
+```bash
+npm install --save ethers
+```
 
-## Installation
+## Installing
 
 ```bash
-// with npm
-npm install @aave/protocol-js
-// with yarn
-yarn add @aave/protocol-js
+npm install --save @aave/protocol-js
 ```
 
-## Usage
+# Data Formatting Methods
+
+*These methods are being updated for v3 with simplified inputs, check back for updated documentation*
```

Comment:
wouldn't put a spoiler as we might end up not doing this :sweat_smile: (although i'm absolutely in favor)